src/ResourceManager/Reservations/Commands.Reservations/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
         - Additional information about change #1
 -->
 ## Current Release
+## Version 0.1.7
```

Comment:
Please remove the `## Version 0.1.7` tag. Just keep the entries under the `## Current Release` tag.

Reply:
Sure. Done.